### DISCUSSION TITLE
how i change notyf default position

### DISCUSSION THREAD

**Discussion by VishiPremWorkz** [php-flasher/php-flasher#140] [Q&A] [answered]
I want to change the default position of notfy flash message from bottom-right to top-right.

return redirect()->route('AdminDashboard')->with('success', 'Welcome Back !!');

I tried this setup in flasher.php but its not working. any suggestion ?
![image](https://user-images.githubusercontent.com/50229405/232011754-031baa02-d417-4002-9364-4f82282866fe.png)

**Comment by yoeunes** [comment]
Hello @VishiPremWorkz, thanks for using PHPFlasher.

Just publish the configuration : `php artisan flasher:install`

Then update `config/flasher_notyf.php` configuration with the same options you provided

**Reply by VishiPremWorkz** [reply]
thanks for reply.
i already publish the config with php artisan flasher:install
then put that config in flasher.php
now i remove that form flasher.php and put that on flasher_notyf.php file.
![image](https://user-images.githubusercontent.com/50229405/232016390-6ca5cfc8-7460-493a-9c02-5c757c9e4f36.png)
still not working..

**Comment by yoeunes** [comment, marked as answer]
I'm sorry for the confusion on this, here is the content of the `config/flasher_notyf.php`

```php
return [
    'scripts' => [
        'cdn' => [
            'https://cdn.jsdelivr.net/npm/@flasher/flasher-notyf@1.2.4/dist/flasher-notyf.min.js',
        ],
        'local' => [
            '/vendor/flasher/flasher-notyf.min.js',
        ],
    ],
    'options' => [
        'position' => [
            'x' => 'right',
            'y' => 'top',
        ],
   ],
];
```

**Reply by VishiPremWorkz** [reply]
Thanks its working..
